### DISCUSSION TITLE
Projects using FSC-PS

### DISCUSSION THREAD

**Discussion by FH-Inway** [fscpscollaborative/fscps#79] [Show and tell]
To get to know FSC-PS, it is nice to see a real live project in action. While most projects are behind closed doors, there are a couple of community projects that use it:
- [D365FOAdminToolkit](https://github.com/ameyer505/D365FOAdminToolkit) by @ameyer505
- [D365FinanceToolbox](https://github.com/jofme/D365FinanceToolbox) by @jofme
- [MXTXppInterpreter](https://github.com/milnet92/MXTXppInterpreter) by @milnet92
- [FO_Integrations](https://github.com/ITRasmus/FO_Integrations) by @ITRasmus
- [ISM-Telemetry-for-Finance-and-Operations](https://github.com/microsoft/ISM-Telemetry-for-Finance-and-Operations) by @Hax1337

Are there other projects out there using FSC-PS? Or do you want to add it to your project? Let us know.

**Comment by jofme** [comment]
You can add https://github.com/ITRasmus/FO_Integrations to the list (-:

**Reply by FH-Inway** [reply]
Thanks, great to see this spreading.
Looking forward to your DynamicsMinds session on how to use GitHub in place of Azure DevOps.

**Reply by jofme** [reply]
Great! See you in Portorož (-:
